### ISSUE TITLE
Add boundaries to historical data push

### ISSUE THREAD

**Issue by JeroenBoersma** [signalise-io/magento2-plugin#3]
It would be awesome that the historical data push could have some boundaries.

Currently you can send a specific order or all orders. Which can be a lot.

Maybe add the following arguments
- `created-before={DateTimeInterfaceInput} (optional)` only process orders created before this date (UTC)
- `created-after={DateTimeInterfaceInput} (optional)` only process orders created on/after this date (UTC)
- `store={storeCode|storeId} (optional)` only process orders from a specific store

Tricky part is here, make sure to have a good convention about times, UTC would be mine.

If you use DateTime(Immutable) as a base you can also set relative times:
https://www.php.net/manual/en/datetime.formats.relative.php

If you combine this you can set a subset of orders to signalise.
```bash
# send all order for last month till yesterday
bin/magento signalise:push-orders --created-before="yesterday" --created-after="last month"

# send all orders from a specific store
bin/magento signalise:push-orders --store=default

# send all order for last year in a specic store
bin/magento signalise:push-orders \
  --created-before='last day of last year 23:59:59' \
  --created-after="first day of last year 00:00:00" \
  --store=nl

```

**Comment by peterjaap** [comment]
I got this as feedback as well at MC22ES!

**Comment by peterjaap** [comment]
I copied @Ndottens his code from #7 into the latest version (I just changed `created-before` / `created-after` to `before` / `after`); https://github.com/signalise-io/magento2-plugin/pull/14

**Comment by peterjaap** [comment]
Available in 1.0.10
